README.md

```diff
@@ -1,6 +1,6 @@
 # FHIR Tools for Anonymization
 
-[![Build Status](https://microsofthealthoss.visualstudio.com/FhirAnonymizer/_apis/build/status/CI%20Build?branchName=master)](https://microsofthealthoss.visualstudio.com/FhirAnonymizer/_build/latest?definitionId=19&branchName=master)
+[![Build Status](https://microsofthealthoss.visualstudio.com/FhirAnonymizer/_apis/build/status/CI%20Build?branchName=master)](https://microsofthealthoss.visualstudio.com/FhirAnonymizer/_build/latest?definitionId=23&branchName=master)

```

Comment:
where does this definitionId come from?

Reply:
This is auto generated pipeline ID. The older pipeline with definitionId 19 is linked to microsoft/fhir-anonymizer repo and the trigger has be inactive as we changed our repo name. I just created a new pipeline with definitionId 23 of the same name 'CI Build'. That's why I need to replace this link.